app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
```

Comment:
This PR is already big as we are starting keywords in this PR. I suggest adding cronjob and HTTP requesting stuff in a separate PR so that it can maintain following standard

- Keeping PR small
- One PR to resolve one specific task
- Smaller PR is easier to review

What do you think?

Reply:
Yes, will move to another PR

Reply:
Fixed in [`f84cf8d` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/f84cf8dbc89dd5a6e5ac9bb1503f04ae0923f7c4)